packages/tpetra/core/src/Tpetra_Details_packCrsGraph_def.hpp

```diff
@@ -715,12 +715,34 @@ packCrsGraph
   }
```

Comment:
I believe all the fences in this file can safely be removed.

Reply:
Removed #7545